04-onion-routing.md

```diff
@@ -275,10 +285,10 @@ The writer:
   - For every node:
     - MUST include `amt_to_forward` and `outgoing_cltv_value`.
   - For every non-final node:
-    - MUST include `short_channel_id`
+    - MUST include `short_channel_id` or `outgoing_node_id`
```

Comment:
LDK's current plan is to support only scids for regular forwards and only node ids for trampoline forwards. I think this is what Acinq does as well. Maybe this should be codified in the spec, if that ends up being everyone's plan? :)

Reply:
Definitely! I'll need to do a big clean-up on this PR, now that LDK is working on it it may be the right time ;)